spec/rubocop/cop/rspec/message_chain_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe RuboCop::Cop::RSpec::MessageChain do
+  subject(:cop) { described_class.new }
+
+  it 'finds `receive_message_chain`' do
+    expect_violation(<<-RUBY)
+      before do
+        allow(foo).to receive_message_chain(:one, :two) { :three }
+                      ^^^^^^^^^^^^^^^^^^^^^ Avoid stubbing using `receive_message_chain`
+      end
+    RUBY
+  end
+
+  it 'finds old `should_receive` syntax' do
```

Comment:
Sorry @bquorning just noticed that this should probably read `stub_chain` not `should_receive`. I'll merge after this is fixed